src/Controls/tests/TestCases.HostApp/Issues/Issue30403.cs

```diff
@@ -0,0 +1,26 @@
+namespace Maui.Controls.Sample.Issues;
+
+[Issue(IssueTracker.Github, 30403, "Image under WinUI does not respect VerticalOptions and HorizontalOptions with AspectFit", PlatformAffected.UWP)]
+public class Issue30403 : TestContentPage
+{
+	protected override void Init()
+	{
+		Title = "Issue 30403";
+
+		Content = new Grid
```

Comment:
Could modify a little bit this test? Could you include a StackLayout with 2-3 images?
Leave the current one, but could also validate:
- Image with extreme aspect ratio
- Really small image

Reply:
Created a separate issue to create the tests https://github.com/dotnet/maui/issues/31686